src/state/reducers.js

```diff
@@ -0,0 +1,40 @@
+import { combineReducers } from 'redux';
+import query from '../advanced-search/state/reducers';
+import search from '../results/state/reducers';
+// import { COPY_QUERY_CLAUSES_OBJECT_TO_SEARCH } from './actions';
```

Comment:
Do you want to keep a reference to these or can we remove them?

Reply:
I used them in the end.